File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_Reports.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_Reports } from './Sys_Reports.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_ReportsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Report_Id', 'Report_Code', 'Report_Name', 'Report_File', 'PaperSize_Id', 'Landscape_Orientation', 'AllowPreview', 'AllowPrintOut', 'Pull_ReportData', 'Package_Name', 'Procedure_Name', 'ReportFormNumber', 'ReportFormVersion', 'ReportFormEffectedDate', 'Enabled', 'Creation_Date', 'Created_By', 'Last_Update_Date', 'Last_Updated_By', 'Attribute1', 'Attribute2', 'Attribute3', 'Attribute4', 'Attribute5', 'Attribute6', 'ExportToDatabase', 'ExecDirectToDatabase', 'IsMultiSiteReport', 'DefaultSiteCode'];

    List: Sys_Reports[] | undefined;
    ListFilter: Sys_Reports[] | undefined;
    FormData!: Sys_Reports;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_Reports";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_ReportsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_ReportsService : BaseService<Sys_Reports, ISys_ReportsRepository>
    , ISys_ReportsService
    {
    private readonly ISys_ReportsRepository _Sys_ReportsRepository;
    public Sys_ReportsService(ISys_ReportsRepository Sys_ReportsRepository) : base(Sys_ReportsRepository)
    {
    _Sys_ReportsRepository = Sys_ReportsRepository;
    }
    }
    }

